Services/Certificate/classes/class.ilCertificateGUI.php

```diff
@@ -532,8 +543,13 @@ private function saveCertificate(ilPropertyFormGUI $form, array $form_fields, $o
                 $backgroundImagePath = $previousCertificateTemplate->getBackgroundImagePath();
 
                 if ($backgroundImagePath === '' && $backgroundImagePath !== null) {
```

Comment:
if `$backgroundImagePath === ''` evals to `true`, `$backgroundImagePath !== null` will be always `true`as well.

Reply:
Yea, this is the old behavior. This isn't the only stupid thing.

I'll fix it